File: modul-326/bomberman-modul-326/unity/Bomberman_Returns/Assets/Models/SendMessages/MovePlayer.cs
using System.Collections;
using System.Collections.Generic;

[System.Serializable]
public class MovePlayer : ClientMessage
{
    public Direction direction;

    public MovePlayer(string playerName, Direction direction) : base(playerName)
    {
        this.direction = direction;
    }

    public Direction Direction { get => direction; }
}

File: modul-326/bomberman-modul-326/src/main/java/ch/gibb/bomberman/protocol/client2server/MovePlayer.java
package ch.gibb.bomberman.protocol.client2server;

import ch.gibb.bomberman.protocol.Direction;

public class MovePlayer extends ClientMessage {
  private Direction direction;

  public MovePlayer(String playerName, Direction direction) {
    super(playerName);
    this.direction = direction;
  }

  public Direction getDirection() {
    return direction;
  }
}
